### ISSUE TITLE
Nazareth Recipe's Vision

### ISSUE THREAD

**Issue by m4v15** [foundersandcoders/nazareth-recipes#47]
There have been multiple discussions of late about the aims and direction of Nazareth recipes, and I think we need to formalise these to guide us forward and prevent any disagreements in the future. 

From [wikipedia](https://en.wikipedia.org/wiki/Vision_statement):
>A vision statement is a company's road map, indicating both what the company wants to become and guiding transformational initiatives by setting a defined direction for the company's growth. 

So, any ideas?

#44 #43 [similar issue on a rival repo](https://github.com/foundersandcoders/gaza-delights/issues/2)

Edit 21:39 5/11/2017 to add a suggested vision
---

>To develop an open source, co-operatively produced hub for communal Founders & Coders recipes. Most notably, those cooked in the Nazareth guesthouse as this is where the majority of communal cooking within Founders & Coders occurs. However contributions from the entire community is both welcomed, and actively encouraged.

**Comment by Karyum** [comment]
I think at first it did start as dishes that were consumed in the guesthouse in Nazareth.

But if we want to expand it and get people's attention we might need to consider writing down a mission statement, IMO we should open it to the entire organization cause it doesn't feel like a FAC thing to do, to just make this repo accessible to only people in Nazareth. but here comes the problem where people need to test the dishes, and it's only here in Nazareth where people get together and have dinner together, and only these people can review the pull request cause only they have tried it and can confirm it.

*this comment was transferred* from #43

**Comment by shiryz** [comment]
Agreeing with @Karyum, I believe this should be a repo for fac's communal meals, the vision should include bringing members together to have delicious food!

**Comment by m4v15** [comment]
So should Nazareth Recipes be a repo to hold all the communal recipe's within FAC? So why should it be called Nazareth Recipes?

I think the way works now is that because in Nazareth we are always cooking for each other, people can make PRs that other people within the community who ate the food can have relevant input in. People cook for each other 4-5 times a week in Naz, but pretty much only 2-3 times a *cohort* elsewhere. So I guess we can say this is mainly about recipes within Nazareth as that is where it came from, but other, _communally_ cooked meals can be added from the wider FAC community

**Comment by m4v15** [comment]
When they make a documentary about the most successful, open source, co-operatively produced hub for recipes online, this issue is going to play a huge role. Be part of history people!

**Comment by finnhodgkin** [comment]
@matthewdking made an interesting suggestion: the wider community should feel free to make PRs with new recipes, but those PRs should not be merged until the meal is cooked by someone at the guesthouse. 

That way we could maintain the original plan for the repo, while also potentially gaining a healthy backlog of community recipes.

**Comment by Karyum** [comment]
@finnhodgkin that's a great idea. it would also encourage people in the guesthouse to cook, and try new recipes that would be awesome!!

we should make it clear though that everyone can raise an issue for a recipe and make pull request but can't be reviewed until it was cooked

**Comment by Karyum** [comment]
should be documented in the README 
rel #44

**Comment by m4v15** [comment]
@Karyum are you saying the vision should be documented in the readme? I was kind of under that assumption anyway I guess. Or this new idea? But yeah, both.

With regards to @matthewdking's suggestion I do like it a lot, we would need guidelines on what to name the PR (they should all be PENDING) or something, maybe give them labels?

**Comment by m4v15** [comment]
However, though I do like the idea, lets not get away from the broader question of what is the vision - this suggestion implies @matthewdking you have some ideas of what you see as the vision? and @finnhodgkin

**Comment by finnhodgkin** [comment]
I was just replying to the discussion above about community participation. I think I'll have to sleep on the broader topic.

**Comment by m4v15** [comment]
@finnhodgkin Fair, I think a considered response is much more valuable and appreciate you approaching the issue with the full seriousness it deserves. We will await your contribution with bated breath.

**Comment by m4v15** [comment]
@/all I have updated the initial comment with the following initial suggestion for a Vision to give us something to work from:
>To develop an open source, co-operatively produced hub for communal Founders & Coders recipes. Most notably, those cooked in the Nazareth guesthouse as this is where the majority of communal cooking within Founders & Coders occurs. However contributions from the entire community is both welcomed, and actively encouraged.

**Comment by Karyum** [comment]
@m4v15 I would imagine Nazareth recipes as a huge library of recipes or strive to be atleast.
I currently think of this repo as a place where i can take a time off coding and serious stuff like mentoring/studying/working.

Just visit this repo watch serious github workflow with silly topics :stuck_out_tongue_winking_eye:  , for example the forzen chocolate cake PR discussing what is an English tray was probably *one* of the funniest discussions i had on github :laughing: .
So as much as it's a place where we can store recipes, it's also a great place to learn github workflow as @finnhodgkin and @matthewdking mentioned and an exit for people who are just bored or want to have a bit of fun

**Comment by shiryz** [comment]
> To develop an open source, co-operatively produced hub for communal Founders & Coders recipes. Most notably, those cooked in the Nazareth guesthouse

This looks like a good start, the rest feels like it belongs to the `contributing` more than it does the vision.

**Comment by mattlub** [comment]
A lot of agreement from me here with @Karyum and @shiryz. My key principles for this repo:
- fun
- wider community interaction

I think we are already achieving high levels of fun, but I think community interaction could be improved; here are a couple of suggestions for that:
- **encourage recipes from anyone in the community**- I believe this is important to engage those out of Nazareth. I know this has been discussed above and others disagree with me so maybe it warrants its own issue?
- **actively encourage comments** in a comments section on recipes (like this empty one [here](https://github.com/foundersandcoders/nazareth-recipes/blob/master/recipes/tagine.md#comments) :wink:) so you can submit a PR and contribute to the repo in a small way without having added a recipe.

**Comment by m4v15** [comment]
@mattlub is there something you would change in the Vision Statement I posted to reflect those?

I think we need to get an overall vision sorted ASAP and then we can sort out how to achieve it later - so the vision can have something about engaging the wider community, but doesn't have to say explicitly about having comments.

@shiryz I know what you mean, the original is a bit wordy, but to reflect the calls for wider community participation we need a bit more, so maybe:

> To develop an open source, active, and co-operatively produced hub for communal Founders & Coders recipes. Most notably, those cooked in the Nazareth guesthouse, ***however*** contributions from the entire community is both welcomed, and actively encouraged 🍽  🥘  🎉 .

@mattlub does this cover your points (emoji's === fun, right?)

**Comment by shiryz** [comment]
@m4v15 I think that all of it should exist in the readme but perhaps our mission could include "encouraging the wider community to contribute to reach ..."
